website/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/local/bin/python3"
+}
```

Comment:
new line

Reply:
This should not be added to the commit as that is my personal vs code linter. I'll look into updating the git ignore.